### ISSUE TITLE
ID: LG-254: Console.log should have a date/time timestamp

### ISSUE THREAD

**Issue by pgunapal** [OpenLiberty/docs#809]
Epic: https://github.com/OpenLiberty/open-liberty/issues/9606

- A new consoleFormat logging server configuration option `simple` will be added to format the messages in the console.log to look like the ones from messages.log, such as with timestamp, threadID, and etc.

- The default message format name was changed to `simple`. All occurrences of this must be updated in the docs.

- The default console format name was changed to `dev`. All occurrences of this must be updated in the docs.

- All occurrences of the format default name `basic` must be removed, from all the documentations mentioning consoleFormat/messageFormat.

In Summary:
consoleFormat: `dev, simple, json`
messageFormat: `simple, json`

The following documentation for Open Liberty should be updated with the appropriate details, above:
https://openliberty.io/docs/ref/general/#logging.html

The following documentation will automatically generated from the metatype information from the code (No action needed for this):
https://openliberty.io/docs/ref/config/logging.html

**Comment by Charlotte-Holt** [comment]
Work complete in https://draft-openlibertyio.mybluemix.net/docs/20.0.0.11/log-trace-configuration.html.
